chapter8.adoc

```diff
@@ -30,8 +69,64 @@ security manager using secure memory.
 When `medbgen` is 1, there are no restrictions. When `medbgen` is 0 but
 `MSDCFG.sdedbgalw` is 1, then the external debug mode may be entered but is
 restricted by `M-mode` software to prevent privileged CSR accesses and memory
-accesses by instructions executed in external debug mode cannot use `M` privilege.
-When `medbgen` is 0 and `MSDCFG.sdedbgalw` is also 0, the M-mode root domain
-security manager must not configure triggers with `action=1`. Triggers for the
-supervisor domain are expected to be controlled by the root domain security
-manager to prevent any leakage of information.
+accesses by instructions executed in Debug Mode cannot use `M-mode`
+privilege. When `medbgen` is 0 and `MSDCFG.sdedbgalw` is also 0, the `M-mode`
+root domain security manager must not configure triggers with `action=1`.
```

Comment:
Even if the triggers are configure with `action=1`, it does yield any side effect, because it just cannot enter Debug Mode. The restriction her  seems redundant.

Reply:
removed this restriction and clarified with the enforcement that M-mode software should ensure that filter mcontrol.m bit should be clear.

Reply:
"clarified with the enforcement that M-mode software should ensure that filter mcontrol.m bit should be clear." This restriction should not be made. It will de-feature self-hosted debug for machine mode

Reply:
not really - this is a software policy completely under m-mode control - don't see how it de-features self-hosted debug at all.